test/test_build.py

```diff
@@ -39,6 +39,43 @@ class ImageBuildResult(NamedTuple):
     metadata: dict = {}
 
 
+def get_distribution_and_version(test_vm, build_ansible_container, username, password):
+    """Run Ansible playbook to get distribution and version"""
+
+    output = test_vm.execute_ansible(build_ansible_container,
+                                     "./test/ansible-playbooks/get_distro_version.yml",
+                                     username,
+                                     password)
+
+    print(output)
+
+    # Parse output to extract distribution and version
+    distribution = None
+    version = None
+    dist_key = "Distribution:"
+    version_key = "Version:"
+    for line in output.splitlines():
+        if dist_key in line:
+            distribution = line.split(dist_key)[1].strip(" \"'")
+        elif version_key in line:
+            version = line.split(version_key)[1].strip(" \"'")
+
+    return distribution, version
+
+
+def check_user(test_vm, build_ansible_container, username, password, user_to_test):
```

Comment:
The function is called `check_` and the doc string says it `gets ...`. I wonder if the function should be called `get_user` instead?

Reply:
thanks for the detailed review - the doc string is wrong